openssl/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+SSL_URL="https://www.openssl.org/source/openssl-3.0.5.tar.gz"
+rm -rf SSL-*
```

Comment:
Please add a cleanup step here.
The `rm -rf SSL-*` command does not do anything since there is nothing in the current directory to match that name.
It should remove the extracted source code, see [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/build.sh#L4) as an example.